gen/common/AArch64Arch_gen.ml

```diff
@@ -210,6 +202,60 @@ let  plain = Plain None
 
 type atom = atom_acc * MachMixed.t option
 
+module Value = struct
+
+  include Value.Make(struct
+    (* Do this on purpose since there is an outside `atom` type. *)
+    type _atom = atom
+    type atom = _atom
```

Comment:
You can use the `nonrec` keyword to have this refer to the `atom` already in scope.
```suggestion
    type nonrec atom = atom
```